pyproject.toml

```diff
@@ -0,0 +1,37 @@
+[build-system]
+requires = ["setuptools>=40.8.0", "wheel"]
+
+[project]
+name = 'bisum'
+version = '0.1.1'
```

Comment:
Right now, the version must be updated in the pyproject.toml _and_ the setup.py file. 

I think we can just get rid of setup.py but we need to see how to add the README/long description in pyproject.toml. That can be a different issue/PR.

Reply:
Just removed setup.py, so only pyproject.toml remains relevant.